src/js/views/patients/patient/data-events/action-item.hbs

```diff
@@ -0,0 +1,9 @@
+    <td class="table-list__cell w-40">
+      <span class="patient__action-icon">{{far "file-alt"}}</span>{{ name }}
+    </td>
+    <td class="table-list__cell w-60">
+      <span class="table-list__meta" data-state-region></span>
+      <span class="table-list__meta" data-owner-region></span>
+      <span class="table-list__meta" data-due-region></span>
+      <span class="patient__action-ts">{{formatMoment updated_at "TIME_OR_DAY"}}</span>
+    </td>
```

Comment:
couplea nits to pick: does this need to be indented? And also should we throw in `{{~ remove_whitespace ~}}` like what we have in the `flow-item` template?